series/series.go

```diff
@@ -22,6 +22,7 @@ const (
 var (
 	// TODO(katco): Remove globals (lp:1633571)
 	seriesOnce sync.Once
+	MustHostSeries = mustHostSeries
```

Comment:
Please have it in a separate block from un-exported vars. Descriptive comment is missing too